content/c/concepts/enums/enums.md

```diff
@@ -0,0 +1,126 @@
+# Enum
+
+A **enum** (enumeration) is a user-defined data type in C. It is used to assign names to integral constants, the principal purpose of this names is to create a program that is easy to read and maintain. It is defined using the `enum` keyword.
+
+## Syntax
+
+```pseudo
+enum name
+{
+	memberName1 = <integral constant>,
+	memberName2,
+	memberName3,
+	...
+	memberNameN
+};
+```
+
+An enum is made up of members where each `memberName` can have a defined integral constant (i.e. 0, 10, 2) or just the `memberName`.
+
+## Example
+
+The following example uses the `enum` keyword to create a `day` enumeration:
+
+```c
+enum day {
+	Sunday, 
+	Monday,
+	Tuesday,
+	Wednesday,
+	Thursday,
+	Friday,
+	Saturday
+};
+```
+
+`Sunday` will be setvalue 0, `Monday` will be set to a value of 1, and so  `Patient` structure.
+
+## Another Example
```

Comment:
You don't need these two examples. The second example below illustrates the same thing. I'd remove these first two, and move that second example below so it's the first example.